schematic/schemas/data_model_nodes.py

```diff
@@ -155,7 +155,9 @@ def run_rel_functions(
             )
 
         elif rel_func == parse_validation_rules:
-            return parse_validation_rules(attr_relationships[csv_header])
+            rules = attr_relationships[csv_header]
```

Comment:
Im clearly worried about people submitting odd things, but since we are gating here we won't hit a typing error if a string is submitted somehow, Can we raise an exception in an else?

Reply:
So I tried this at one point:

```
elif rel_func == parse_validation_rules:
    rules = attr_relationships[csv_header]
    if  isinstance(rules, dict) or isinstance(rules, list):
        return parse_validation_rules(rules)
   else:
      raise ValueError
```
but a bunch of tests failed. The way I have it now is closest to the current behavior, as there were not any exceptions raised before.